spec/models/book_spec.rb

```diff
@@ -58,4 +58,41 @@
       end
     end
   end
+
```

Comment:
again looks really good, i like mixing the order of the books as you create them and adding an extra one to make sure its limit 3, all really easy to read and test for edgecasing